build/Dockerfile

```diff
@@ -16,7 +16,10 @@ FROM registry.access.redhat.com/ubi8/ubi:latest
 
 ENV OPERATOR=/usr/local/bin/eunomia-operator
 
+# include template files as default ones
+COPY deploy/helm/eunomia-operator/job-templates/* /default-templates/
```

Comment:
With removal of the templates/configmap.yaml file, it looks to me like the files in `job-templates/` are no longer used by the helm infrastructure in `deploy/helm/eunomia-operator/`, no? If yes, could we move the `job-templates/` dir into the `build/` dir?

Reply:
That's a good idea